gooddata-sdk/gooddata_sdk/client.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         self._layout_api = apis.LayoutApi(self._api_client)
         self._actions_api = apis.ActionsApi(self._api_client)
         self._user_management_api = apis.UserManagementApi(self._api_client)
+        self._filter_views_api = apis.FilterViewsApi(self._filter_views_api)
```

Comment:
I believe that this is not necessary. `FilterViewsApi` should be part of `self._layout_api ` or `self._entities_api` or are they not? There is `self._user_management_api` because these are use case api.